wild-swimming-app/src/components/add-comment-form/Addcommentform.tsx

```diff
@@ -0,0 +1,79 @@
+import { postComments } from "./utils/addcomments";
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { useAuth0 } from "@auth0/auth0-react";
+
+
+
+const AddCommentForm = () => {
+  const { user, isAuthenticated} = useAuth0();
+  console.log(user);
+  const params = useParams();
+  const location_Id =params.id;
+
+const[userName, setUserName]= useState('water_baby')
+const[submitting, setSubmitting]= useState(false);
+const[bodyMessage, setBodyMessage]= useState("")
+const [err, setErr] = useState("");
+const[commentsList, setCommentsList]= useState("");
+
+const[message, setMessage]= useState("");
+
+
+const handleSubmit = (event:any)=>{
+  event.preventDefault();
+    const inputComment ={
+     name:userName,
```

Comment:
great use of camel case, as they won't have usernames but the user does have a Name!

Reply:
yes, i've not managed to add that from  the log in info but i will update when I do